### ISSUE TITLE
Error Loading Plugin

### ISSUE THREAD

**Issue by anshumanbhatia** [asednev/homebridge-plugin-govee#24]
**Describe Your Problem:**
The plug is no longer working after the upgrade to 1.3.3

**Logs:**

```
 ERROR LOADING PLUGIN homebridge-plugin-govee:
[3/11/2021, 8:43:47 AM] TypeError: require(...) is not a constructor
    at module.exports (/usr/local/lib/node_modules/homebridge-plugin-govee/node_modules/@abandonware/noble/lib/resolve-bindings.js:11:12)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge-plugin-govee/node_modules/@abandonware/noble/index.js:2:51)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge-plugin-govee/node_modules/govee-bt-client/dist/index.js:17:33)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
[3/11/2021, 8:43:47 AM] ====================
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.

    {
            "name": "Govee",
            "batteryThreshold": 25,
            "debug": true,
            "humidityOffset": 0,
            "platform": "GoveeHomebridgePlugin"
        }
```

**Screenshots:**

**Environment:**

* **Plugin Version**: 1.08? Even though I just did the upgrade to 1.1?
* **Homebridge Version**:  1.33
* **Node.js Version**:  14.16
* **NPM Version**:  6.4.11
* **Operating System**:  MacOs Mojave

**Comment by asednev** [comment]
Thank you for reporting this issue. I am looking into this and hope to have a fix in the next 1-2 days. I suggest trying to install an older version in the mean time.

**Comment by asednev** [comment]
@anshumanbhatia this should be fixed in version 1.0.12 of the plugin.

**Comment by anshumanbhatia** [comment]
Amazing work. Thank you!
